src/site/apt/usage.apt.vm

```diff
@@ -74,7 +74,7 @@ Configuration
   The reports will link directly to the cross-referenced source if you enable this with the <<<linkXRef>>> parameter.
   See the {{{http://maven.apache.org/plugins/maven-jxr-plugin/}JXR plugin}} for more details.
 
-  If your source uses a non-default encoding, you can use the <<<sourceEncoding>>> parameter to tell Maven which
+  If your source uses a non-default encoding, you can use the <<<inputEncoding>>> parameter to tell Maven which
   encoding to use when reading the java source.  Note also the ability to exclude source which you want
   to ignore.
```

Comment:
why such change?
Is it not braking?

Reply:
This basically as in https://github.com/apache/maven-checkstyle-plugin/pull/87#discussion_r950316588. We would end up in two params for encoding.